src/Propel/Common/Config/PropelConfiguration.php

```diff
@@ -27,14 +27,13 @@ class PropelConfiguration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $isBeforeSymfony5 = method_exists(TreeBuilder::class, 'root');
```

Comment:
Honestly, I prefer the documenting variable over the comment you added. Nobody reads comments. Do you mind restoring this as something like
```
$isSymfony5OrLater = `method_exists($treeBuilder, 'getRootNode');
if($isSymfony5OrLater){
    ...
```

Reply:
Yes, I can change the code to using a variable if you prefer it. But the variable would be `$isSymfony42OrLater`.

Reply:
Maybe `$isAtLeastSymfony42`? I really think it is much better to use variable names instead of comments, but I am happy to shut up about it if other reviewers dissagree